### ISSUE TITLE
abortOrigin of undefined

### ISSUE THREAD

**Issue by anasanvila** [avocode/avocode#2140]
---
* **Version**: 3.4.1
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 411006
---
```
TypeError: Cannot read property 'abortOrigin' of undefined
    at t._handleChildAbort (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:80490)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:77655
    at normalize (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/ljson/LJSON.js:64:44)
    at normalize (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/ljson/LJSON.js:81:78)
    at normalize (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/ljson/LJSON.js:81:78)
    at normalize (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/ljson/LJSON.js:81:78)
    at normalize (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/ljson/LJSON.js:73:48)
    at normalize (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/ljson/LJSON.js:81:78)
    at normalize (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/
```

**Comment by jakubzitny** [comment]
Hi @anasanvila, thanks for reporting this problem. It's a bug, we can see it in our internal tracking system and we'll try to fix it as soon as possible.

Can you please give us more details about this? Did you try it more than once? Are you unable to upload any designs to Avocode altogether? How did you try to do it? You can upload by drag+dropping but you can also use the File menu -> Upload design option. You can also [use our Photoshop plugin](https://help.avocode.com/projects-and-designs/design-upload/how-to-sync-psd-designs-to-avocode-via-photoshop-plugin). I think when you try again, upload should work correctly, because this issue should be happening only very occasionally. We'll try to fix it though.

**Comment by jakubzitny** [comment]
Hi @anasanvila, thanks for reporting this problem. It's a bug, we can see it in our internal tracking system and we'll try to fix it as soon as possible.

Can you please give us more details about this? Did you try it more than once? Are you unable to upload any designs to Avocode altogether? How did you try to do it? You can upload by drag+dropping but you can also use the File menu -> Upload design option. You can also [use our Photoshop plugin](https://help.avocode.com/projects-and-designs/design-upload/how-to-sync-psd-designs-to-avocode-via-photoshop-plugin). I think when you try again, upload should work correctly, because this issue should be happening only very occasionally. We'll try to fix it though.

**Comment by jakubzitny** [comment]
Hi @anasanvila, we've just released `3.5.1` and this problem is fixed.

Let us know if you have any more problems 😉
